ANMN/burst_averaged_product/generate_wqm_nc_file_gatt

```diff
@@ -0,0 +1,4 @@
+[global_attributes]
```

Comment:
Same here. Don't hardcode abstract and keywords. What you add should be the same for WQM and CTD.